experiments/conductor/cmd/runner/scripts/mockgcp/02-run-script-real-gcp.sh

```diff
@@ -62,7 +62,7 @@ echo "Running test"
 # HACK: refresh token
 gcloud auth print-access-token > /dev/null
 # We ignore test failures, because we expect the golden output to be volatile at this stage
-(go test ./mockgcptests -v -run TestScripts/${RUN_TEST} || true) | tee ${LOG_DIR}/test-realgcp.log
```

Comment:
1 hr 20 minutes, seems a bit long for an timeout for mockgcptests. Wondering if we've seen a successful run which was anywhere near that long?

Reply:
This script is used to run the real gcp test. For Cloud Composer, it takes 25 minutes to create the environment, a few minutes to a couple of hours (it depends) to be updated, and 10 to 40 minutes to be deleted. So, I'm using 4800s just to be able to finish all the gcloud commands.

Reply:
Ouch!  A mock is going to be a huge time-saver .... once it's done :-)